### DISCUSSION TITLE
Give us back the custom zoom option (Z keybind/hotkey) from ImageGlass 8 when slideshow mode is active

### DISCUSSION THREAD

**Discussion by GGuyX** [d2phap/ImageGlass#2017] [General] [answered]
My biggest gripe with upgrading from ImageGlass 8 to 9 has got to be the major changes done to how slideshow mode functions. I have both: Highly preferred the old slideshow functionality in ImageGlass 8 **and** what's more important and relevant to my suggestion, I have loved that we were able to use custom zoom values during our slideshows by simply pressing Z key when they were active. Now, because of the major changes that I have read when it comes to how slideshow mode works between ImageGlass 8 and 9, I know that this functionality was stripped away from slideshows, however I am asking for one, simple thing and one thing only. And that is: Can we have that back? The custom zoom functionality during slideshows was very useful for some very specific, albeit highly niche use cases. Not having it is a bit of an problem for how I typically use the software. It's an simple ask because you guys provide the single best image viewer kind software for completely free and so far, nothing else I have used, including the solutions built-in into Windows or those promoted by Microsoft have worked well for me at all, lacking the advanced capabilities and specific tools I needed from them. I am very sorry if this was already suggested before - I did perform an quick search for any similar topics but haven't found any. Thanks for your attention and time to the ImageGlass development team!

Image to easily understand which specific functionality of the program I am referring to:
![image](https://github.com/user-attachments/assets/38a678d5-e7eb-4691-a89e-610142df53dd)

**Comment by d2phap** [comment, marked as answer]
Hi @GGuyX 
I just created a ticket #2021 as feature request. I will implement this in the next release.

**Reply by GGuyX** [reply]
Thank you for this. I'm glad it was considered for adding back :D ❤️

**Reply by d2phap** [reply]
FYI, it's implemented in [ImageGlass 9.2 Beta](https://github.com/d2phap/ImageGlass/releases/tag/9.1.8.1104)
